olm/quickstarts/rhosak-openshift-getting-started-quickstart.yaml

```diff
@@ -130,6 +130,34 @@ spec:
         success: >-
           You have completed this task!
         failed: Try the steps again.
+    - title: Setting permissions for a service account in a Kafka instance in Streams for Apache Kafka
+      description: >-
+        After you create a service account to connect to a Kafka instance, you must also set the appropriate level of access for that new account in the Access Control List (ACL) of the Kafka instance. {product} uses ACLs provided by Kafka that enable you to manage how other user accounts and service accounts are permitted to interact with the Kafka resources that you create.
+
+        1. In the **Kafka Instances** page of the web console, click the name of the Kafka instance that you want the service account to access.
+
+        1. Click the **Access** tab to view the current ACL for this instance.
+
+        1. Click **Manage access**, use the **Account** drop-down menu to select the service account that you previously created, and click **Next**.
+
+        1. Under **Assign Permissions**, use the drop-down menus to set the following permissions for the specified resource types for this service account. Click **Add** to add each new resource permission.
```

Comment:
See this comment on PR #325: https://github.com/redhat-developer/app-services-guides/pull/325/files#r723628737